src/funtracks/import_export/magic_imread.py

```diff
@@ -0,0 +1,136 @@
+"""Adapted implementation of napari.napari_builtins.io._read.magic_imread"""
+
+import re
+from glob import glob
+from pathlib import Path
+
+import dask.array as da
+import numpy as np
+import tifffile
+import zarr
+from dask import delayed
+from numpy.typing import ArrayLike
+
+
+def _alphanumeric_key(s: str):
+    return [int(c) if c.isdigit() else c for c in re.split("([0-9]+)", s)]
+
+
+def _guess_zarr_path(path: Path) -> bool:
+    return any(part.endswith(".zarr") for part in path.parts)
+
+
+def read_zarr_dataset(path: Path) -> tuple[ArrayLike | list[ArrayLike], tuple[int]]:
+    """Read a zarr dataset, including an array or a group of arrays."""
+    if (path / ".zarray").exists():
+        image = da.from_zarr(path)
+        shape = image.shape
+    elif (path / ".zgroup").exists():
```

Comment:
Let's delete this path, since it tries to read all zarr arrays as images. If we pass it a zarr group containing a geff, it will try to read the columns as images 😞